### PR TITLE
Code re-organisation and CSS changes

### PR DESCRIPTION
**Pull request by aashiks**
- Added http-server and watch as dependencies for development ease.
- Added npm start task and related tasks to do RAD
- Added siimple and siimple-colors as dependencies
- Re-organised code a bit.
- Modified the index page layout with CSS changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,12 +7,25 @@
   "author": "Akshay S Dinesh <asdofindia@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "build": "jspm bundle src/ascii-unicode-converter.js dist/bundle.js --skip-source-maps"
+    "run-server": "node_modules/http-server/bin/http-server --cors -p 9000",
+    "postinstall": "cp node_modules/siimple/dist/*.min.css src/assets/ && cp node_modules/siimple-colors/dist/*.min.css src/assets/",
+    "build:css": "cp -R src/assets/ dist/",
```
Comment (asdofindia) [inline package.json]: Why cp to src postinstall and then to dist during build?
Why not directly copy from the real source folder (which's the node_modules) to dist folder during build?

Comment (aashiks) [reply package.json]: postinstall ensures all css required for project ends up in assets. 

Build just depends on computation (jspm and whatever) and its entire input comes from src folder and has no dependency on node_modules.

Thus the only time node_modules (or whatever it is) comes into play is during postinstall, once. The dependencies for build is minimized to src folder. That's how I see it. If you need me to do it another way I'll gladly change it.

Comment (asdofindia) [reply package.json]: Why I'm feeling queasy about it is,

1. I'm used to seeing src folder have contents that is actually authored by the authors of the project.
2. The way this is now, right after `git clone && npm install` a fresh copy of the repository has untracked files.

There are two ways of adding third party code to repos that I've seen and grown comfortable with:
1) Having a `vendor` folder (which could be inside `src` or in root) consisting of third party code, and it's committed into git. This approach is usually used by projects where such dependencies are manually maintained and not frequently updated.
2) Not committing third party code into git at all, rather, using package.json or other dependency manager to keep track of the code. These are then referenced from build scripts and they become part of the generated code.

Comment (aashiks) [reply package.json]: Makes sense. I'll add more commits that fixes it directly using #2. Like I said, my sole interest was minimizing the surface of build. Going this way sort of forces you into a workflow that is reliant on the build as well.

**Pull request merged**